pennylane/qnn/keras.py

```diff
@@ -216,8 +216,12 @@ def __init__(
             self._signature_validation(qnode, weight_shapes)
             self.qnode = to_tf(qnode, dtype=tf.keras.backend.floatx())
 
-        # Allows output_dim to be specified as an int, e.g., 5, or as a length-1 tuple, e.g., (5,)
-        self.output_dim = output_dim[0] if isinstance(output_dim, Iterable) else output_dim
+        # Allows output_dim to be specified as an int or as a tuple, e.g, 5, (5,), (5, 2), [5, 2]
+        # Note: Single digit values will be considered an int and multiple as a tuple
+        if isinstance(output_dim, Iterable) and len(output_dim) > 1:
+            self.output_dim = tuple(output_dim)
+        else:
+            self.output_dim = output_dim[0] if isinstance(output_dim, Iterable) else output_dim
```

Comment:
I actually thought a lot about this and my first instinct was to make `tuple` as the default `output_dim` type but that required major overhauls in the tests and it would also break the existing applications of this when `output_dim` is being accessed outside of the `KerasLayer`. The two things that I kept in mind while making the changes were 1) to not break the current implementations and 2) to make as less changes to the code as possible while adding the support. Hence, I decided to make the possible `output_dim` type to be either an integer or a tuple.

Reply:
Nice, sounds good!